src/main/java/net/snowflake/client/jdbc/RestRequest.java

```diff
@@ -417,7 +428,19 @@ public static CloseableHttpResponse execute(
             logger.debug("sleeping in {}(ms)", backoffInMilli);
             Thread.sleep(backoffInMilli);
             elapsedMilliForTransientIssues += backoffInMilli;
-            backoffInMilli = backoff.nextSleepTime(backoffInMilli);
+            if (isLoginRequest) {
+              backoffInMilli = backoff.getJitterForLogin(backoffInMilli);
```

Comment:
What about taking random value between (backOffInSec + jitter) and (2^retryCount + jitter) ?
As I understand from doc and python implementation, we should have something like this:

- 1 retry 
    sleep = 1s
- 2 retry
    jitter = 0.5 * 1s * chooseRandom(-1,1) -> (-0.5s, +0.5s)
    sleep = chooseRandom(1s + jitter, 4s + jitter) -> (0.5s, 4.5s)
- 3 retry
    jitter = 0.5 * prev.sleep * chooseRandom(-1,1) -> (-2.25s, 2.25s)
    sleep = chooseRandom(prev.sleep + jitter, 8s + jitter) -> (0.25s, 10.25s)

Reply:
I can make that change I was just following what it looked like we were doing for the ODBC driver.

Reply:
It should be similar in all drivers. Please contact @sfc-gh-ext-simba-hx.
One more thing - chooseRandom(-1,1) should randomly return -1 or 1, not the random number between -1 and 1.